playbooks/roles/load-balancer-v2/defaults/main.yml

```diff
@@ -7,8 +7,10 @@ haproxy_domain_regex: "haproxy.*\\.net\\.opencraft\\.hosting"
 haproxy_ppa: "ppa:vbernat/haproxy-1.8"
 haproxy_version: "1.8.*"
 
-haproxy_consul_watch_prefix: "ocim/"
+haproxy_consul_watch_prefix: "ocim/instances"
```

Comment:
I recall there was a reason that the `haproxy_consul_watch_prefix` had to be `ocim/` rather than `ocim/instances` but can't remember the details. I will have to dig that info up to make sure this is okay.

Reply:
@SSPJ, sure. I made this change because Uman had mentioned in that comment that this had to be done. Otherwise I have no prior context on this.

Reply:
@SSPJ, did you find out about why the watch prefix had to be `ocim/` instead of `ocim/instances` as recommended by Uman?